ftw/simplelayout/browser/client/web/js/app/simplelayout/Block.js

```diff
@@ -49,6 +49,13 @@ define([
       return this;
     };
 
+    this.markLinks = function() {
+      var url = window.location.protocol + "//" + window.location.host;
+      // all http links (without the link-plain class), not within this site
+      $("a[href^=\"http\"]:not(.link-plain):not([href^=\"" + url + "\"])", this.element).attr("target", "_blank");
```

Comment:
The `mark_special_links.js` script does more than just setting the `target="_blank"`. It wraps links in `span`. It is not designed to be run multiple times.
I don't think we use those link wrappers anywhere. It looks like this was used for styling. So it is a bit outdated as we can use advanced css nowadays.